database/migrations/2017_04_21_162001_create_vouchers_table.php

```diff
@@ -15,20 +15,16 @@ public function up()
     {
         Schema::create('vouchers', function (Blueprint $table) {
             $table->increments('id'); // a voucher *instance*.
-            $table->integer('assigner_id')->unsigned()->nullable(); // who was assigned this voucher.
-            $table->integer('redeemer_id')->unsigned()->nullable(); // who redeemed this with
+            $table->integer('assignee_id')->unsigned()->nullable(); // who was assigned this voucher.
```

Comment:
nope it was a typo(!). see below about what asignee does (and why it might be pointless)